src/main/java/seedu/jarvis/ui/MainWindow.java

```diff
@@ -357,11 +357,10 @@ public MissionListPanel getMissionListPanel() {
     private String getLoginMessage() {
         StringBuilder sb = new StringBuilder();
 
-        if (!logic.hasUsername()) {
-            sb.append("Please edit your username using `edit -l u/[USERNAME]` and restart to fetch missions/quests");
+        if (!logic.hasUsername() || !logic.hasPassword()) {
```

Comment:
Interesting decision -- may I ask why?

Reply:
Right now, it will show either `edit password` or `edit username` but ideally both should be shown to the user.